docs/machine-learning/tutorials/object-detection-onnx.md

```diff
@@ -0,0 +1,1124 @@
+---
+title: 'Tutorial: Detect objects using deep learning with ONNX and ML.NET'
+description: This tutorial illustrates how to use a pre-trained ONNX deep learning model in ML.NET to detect objects in images.
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 07/10/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a developer, I want to use ML.NET so that I can use a pre-trained model in an object detection scenario to detect objects in images using ONNX.
+---
+
+# Tutorial: Detect objects using ONNX in ML.NET
+
+Learn how to use a pre-trained ONNX model in ML.NET to detect objects. The pre-trained model was trained to find and classify objects in an image.
+
+Training an object detection model from scratch requires setting millions of parameters, a large amount of labeled training data and a vast amount of compute resources (hundreds of GPU hours). Using a pre-trained model allows you to shortcut the training process.
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+> * Understand the problem
```

Comment:
I think we could expand this list to make it a little less generic. They learn a lot in this tutorial!